### ISSUE TITLE
New Spec added: ls_var_tmp 

### ISSUE THREAD

**Issue by xiangce** [RedHatInsights/insights-frontend-assets#11]
from https://github.com/RedHatInsights/insights-core/pull/1062

```
ls_var_tmp = simple_command("/bin/ls -lanR /var/tmp")
```

**Comment by kylape** [comment]
I would think this spec has the potential to be unpredictably large.  If you agree, can we come up with an alternative way to collect the necessary information?

**Comment by TripleJQK** [comment]
@kylape 
about `find` , I am afraid it may trigger unexpected heavy load.

What about `/bin/ls -lan /var/tmp`?

~~~
[root@mail tmp]# for i in `seq 1 10000`; do touch ab12345665234234234324324-$i; done
[root@mail tmp]# ls -lan /var/tmp/ >/tmp/l1
[root@mail tmp]# cat /tmp/l1 |wc -l
10003
[root@mail tmp]# ll -h /tmp/l1
-rw-r--r--. 1 root root 683K Apr 10 07:27 /tmp/l1
~~~
for 10000 lines long file name, it will be 683k

**Comment by TripleJQK** [comment]
update this to

ls_var_tmp = simple_command("/bin/ls -ln /var/tmp")

**Comment by xiangce** [comment]
Fixed in https://github.com/RedHatInsights/insights-core/pull/1103

**Comment by kylape** [comment]
Resolved in ebe8c96

**Comment by xiangce** [comment]
Thanks!
